### PR TITLE
docs(scully): add manual installation instructions

### LINKED ISSUE #878: Create a "manual installation" page/guide

### LINKED ISSUE #1359: Build a Manual Installation Page
### Description

And a manual installation page for adding Scully to your project.

### PR DESCRIPTION
**Pull request by mainawycliffe**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] Other... Please describe: Docs Update 

## What is the current behavior?

Add Manual Installation Instruction For Scully

Closes #1359 and Closes #878

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by mainawycliffe** [comment]
@SanderElias should I add instructions to create the plugins directory as well?

File: docs/learn/getting-started/installation.md [hunk 1]
```diff
@@ -84,3 +84,66 @@ export const config: ScullyConfig = {
...
+  imports: [
+    // ... other modules imports
+    ScullyLibModule,
+  ],
+  bootstrap: [AppComponent],
+})
+export class AppModule {}
+```
+
+**NB:** You can pass configurations to the `ScullLibModule` using the `forRoot`
```
Comment (Jefiozie) [inline docs/learn/getting-started/installation.md:121]:
```suggestion
**NB:** You can pass configurations to the `ScullyLibModule` using the `forRoot`
```

File: docs/learn/getting-started/installation.md [hunk 2]
```diff
@@ -84,3 +84,66 @@ export const config: ScullyConfig = {
   ...
 }
 ```
+
+### Manual Installation
+
+First, install both `@scullyio/scully` and `@scullyio/ng-lib` npm packages
```
Comment (Jefiozie) [inline docs/learn/getting-started/installation.md:90]:
```suggestion
First, install the following packages `@scullyio/scully` and `@scullyio/ng-lib`
```

File: docs/learn/getting-started/installation.md [hunk 3]
```diff
@@ -84,3 +84,66 @@ export const config: ScullyConfig = {
...
+```ts
+export interface ScullyLibConfig {
+  useTransferState?: boolean;
+  alwaysMonitor?: boolean;
+  manualIdle?: boolean;
+  baseURIForScullyContent?: string;
+}
+```
+
+And finally, create a `scully.[PROJECT_NAME].config.ts`, replacing
```
Comment (Jefiozie) [inline docs/learn/getting-started/installation.md:134]:
```suggestion
Finally, create a `scully.[PROJECT_NAME].config.ts`, replacing
```

File: docs/learn/getting-started/installation.md [hunk 4]
```diff
@@ -84,3 +84,66 @@ export const config: ScullyConfig = {
...
+`[PROJECT_NAME]`, with the name of your Angular project. And then add the
+content of the Scully config file like shown below:
```
Comment (Jefiozie) [inline docs/learn/getting-started/installation.md:136]:
```suggestion
`[PROJECT_NAME]`, with the name of your Angular project. Then add the
content of the Scully config file like shown below:
```

**Review by Jefiozie — CHANGES_REQUESTED** [review]
Hi, @mainawycliffe I've made some suggestions to your PR, would you mind providing me with feedback if these are valid changes? If you agree with the changes would you be willing to change them in this PR?

**Comment by mainawycliffe** [comment]
No, problem, I will take care of the changes.

**Comment by SanderElias** [comment]
@mainawycliffe Thanks for writing this! it is well needed. 
Still, I would love it even slightly more if the manual installation was in a separate document. 
A `manualInstallation.md` next to the file where it is now would do quite nicely.

**Comment by mainawycliffe** [comment]
@SanderElias Yeah, that makes sense. Will make the change.

**Review by SanderElias — APPROVED** [review]

**Review by Jefiozie — APPROVED** [review]
LGTM

**Pull request merged**